packages/styles/scss/components/content-group/_content-group.scss

```diff
@@ -0,0 +1,73 @@
+//
+// Copyright IBM Corp. 2016, 2018
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+@import '../../globals/imports';
+
+@mixin themed-items {
+  .#{$prefix}--content-group__title {
+    color: $text-01;
+  }
+}
```

Comment:
I'm seeing this pattern used multiple times now, and I'm a little worried we're not thinking about the scope of our `SCSS`. If I'm not mistaken this is just overwriting the other previously imported mixins we created called `themed-items`. I think we can scope it to this component a little better with a better naming convention like `content-group-theme`, or maybe just include its contents within the mixin below. Also, if we continue down this path does it make sense to break this mixin out into its own file? That way we have a single file handling the theming aspect.

Reply:
@jeffchew do we need to include `carbon--type-style('expressive-heading-04');` in here to align with the new theming strategy? I don't know, would also some of the `box-modal` styling be needed as well?

Reply:
> I'm seeing this pattern used multiple times now, and I'm a little worried we're not thinking about the scope of our `SCSS`. If I'm not mistaken this is just overwriting the other previously imported mixins we created called `themed-items`. I think we can scope it to this component a little better with a better naming convention like `content-group-theme`, or maybe just include its contents within the mixin below. Also, if we continue down this path does it make sense to break this mixin out into its own file? That way we have a single file handling the theming aspect.

Adding @annawen1 here. I think we're ok for now since the theme mixin is getting referenced right away in the same file, but to be on the safe side it might be worth creating different mixin names for the theming. Not sure if it's necessary to break them out into their own file though.

Reply:
Yeah, it's getting referenced right before it's used, but this doesn't allow us to call it again in the future. As an example, if this theme needs to be changed in a different context using `SCSS` we might have some issues.

Here is just a quick example of what I mean...
```scss
@import 'components/scss/component-1';
@import 'components/scss/component-2';

.component-3 {
  @include carbon--theme($carbon--theme--g90) {
    @include themed-items;
  }
}
```

Which `themed-items` are we theming in `component-3`? Based on the order it would be `component-2`, and there is now no way to theme `component-1` if that's what we were wanting to do within this context. We use a similar technique in theming the footer accordion and button.